index.html

```diff
@@ -144,6 +144,59 @@ <h4 class="heading heading_level-4">Подушка</h4>
             </div>
         </section>
 
+        <section class="section-services">
+            <div class="column-wrapper">
+                <ul class="section-services-list">
+                    <li class="section-services-list__item">
+                        <img class="section-services-list__image" src="styles/blocks/section-services/images/support.svg" alt="Поддержка(текст в макете перепутан со значками)">
+                        <p class="section-services-list__text section-services-list__text__support">Круглосуточная служба поддержки</p>
```

Comment:
Здесь и везде ниже. имя модификатора отделяется от имени блока или элемента одним нижним подчеркиванием
```suggestion
                        <p class="section-services-list__text section-services-list__text_support">Круглосуточная служба поддержки</p>
```